sdk/serialization/azure-xml/src/main/java/com/azure/xml/implementation/XmlInputFactoryFacade.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.xml.implementation;
+
+import javax.xml.stream.XMLInputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamReader;
```

Comment:
same comments about the package not being available for android
As you can see, javax.xml.stream is missing entirely https://developer.android.com/reference/javax/xml/package-summary

Reply:
Hmm, this wasn't an issue before, in `com.azure:azure-xml:1.1.0` all these imports were being used in `XmlReader`, and that versions was being used by `com.azure:azure-core` since `1.51.0` (and the same holds true for `com.azure:azure-xml:1.0.0`, which `azure-core` was using since `1.48.0`)

Reply:
https://github.com/Azure/azure-sdk-for-java/blob/azure-xml_1.0.0/sdk/serialization/azure-xml/src/main/java/com/azure/xml/XmlReader.java

Reply:
to be fair our CI is only complaining about `javax.xml.stream.util` in this class `com.azure.xml.implementation.aalto.stax.InputFactoryImpl`

So maybe there's a documentation gap, and the javax.xml.stream is supported on android, just not the util sub-package?